packages/teams-js/package.json

```diff
@@ -12,15 +12,15 @@
   "typings": "./dist/esm/packages/teams-js/dts/index.d.ts",
   "module": "./dist/esm/packages/teams-js/src/index.js",
   "scripts": {
-    "build": "pnpm clean && pnpm lint && pnpm build-rollup && pnpm build-webpack && pnpm docs:validate",
+    "build": "pnpm clean && pnpm lint && pnpm build-rollup && pnpm build-webpack && pnpm docs:validate && size-limit",
```

Comment:
Do you anticipate that the below `size` command will be used in a more complex manner in the future? If not, we could simplify future code changes by calling `pnpm size` in the build script and then only have to update the `size` command below whenever we make an intended change to functionality.

Reply:
I'm not sure I fully understand what you're saying here. Do you mean not running `pnpm size` as part of `pnpm build` and instead running it as a pipeline step in the pipeline?

Reply:
Sorry, I what I meant was why do we have `&& size-build` instead of `&& pnpm size` in the `build` command.

Reply:
Oh yeah, that was an oopsie on my part. Fixed!